extension/commercial_data/README.md

```diff
@@ -272,6 +272,7 @@ Get full metadata of the order. The item should follow the [STAC Order Extension
     {
       "type": "Feature",
       "stac_version": "1.0.0",
+      "stac_extensions": ["proj", "eo", "view", "processing"],
```

Comment:
These are the v0.9 extension names, in v1.0.0 you need the schema URIs. For example `https://stac-extensions.github.io/eo/v1.0.0/schema.json` instead of `eo`. Please check the extension READMEs for the corresponding URIs.

Reply:
Thanks, fixed.